src/nextcloud/api_wrappers/webdav.py

```diff
@@ -102,30 +103,42 @@ def download_file(self, uid, path):
         with open(filename, 'wb') as f:
             f.write(res.data)
 
-    def upload_file(self, uid, local_filepath, remote_filepath):
+        # set timestamp of downloaded file
+        file_timestamp_str = (file_data.data[0].get('last_modified'))
+        try:
+            file_timestamp = datetime.strptime(file_timestamp_str, '%a, %d %b %Y %H:%M:%S GMT')
+            os.utime(filename, (datetime.now().timestamp(), file_timestamp.timestamp()))
+        except ValueError:
+            pass
```

Comment:
Please add a comment explaining when can we except `ValueError` to be raised, and why it doesn't matter.